sql/updates/world/master/2017_11_26_00_world.sql

```diff
@@ -0,0 +1,59 @@
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET NAMES utf8 */;
+/*!50503 SET NAMES utf8mb4 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+
+DROP TABLE IF EXISTS `creature_template_resistance`;
+CREATE TABLE IF NOT EXISTS `creature_template_resistance` (
+  `CreatureID` MEDIUMINT(8) UNSIGNED NOT NULL,
+  `School` TINYINT(6) UNSIGNED NOT NULL,
+  `Resistance` SMALLINT(6) DEFAULT NULL,
+  `VerifiedBuild` SMALLINT(5) DEFAULT '0',
+  PRIMARY KEY (`CreatureID`, `School`)
+) ENGINE=INNODB DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `creature_template_spell`;
+CREATE TABLE IF NOT EXISTS `creature_template_spell` (
+  `CreatureID` mediumint(8) unsigned NOT NULL,
+  `Index` TINYINT(6) unsigned NOT NULL DEFAULT '0',
+  `Spell` mediumint(8) unsigned DEFAULT NULL,
+  `VerifiedBuild` smallint(5) DEFAULT '0',
+  PRIMARY KEY (`CreatureID`, `Index`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+/*!40101 SET SQL_MODE=IFNULL(@OLD_SQL_MODE, '') */;
+/*!40014 SET FOREIGN_KEY_CHECKS=IF(@OLD_FOREIGN_KEY_CHECKS IS NULL, 1, @OLD_FOREIGN_KEY_CHECKS) */;
+/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
+
+INSERT INTO creature_template_resistance SELECT entry, 1, resistance1, VerifiedBuild FROM creature_template WHERE resistance1 <> 0;
+INSERT INTO creature_template_resistance SELECT entry, 2, resistance2, VerifiedBuild FROM creature_template WHERE resistance2 <> 0;
+INSERT INTO creature_template_resistance SELECT entry, 3, resistance3, VerifiedBuild FROM creature_template WHERE resistance3 <> 0;
+INSERT INTO creature_template_resistance SELECT entry, 4, resistance4, VerifiedBuild FROM creature_template WHERE resistance4 <> 0;
+INSERT INTO creature_template_resistance SELECT entry, 5, resistance5, VerifiedBuild FROM creature_template WHERE resistance5 <> 0;
+INSERT INTO creature_template_resistance SELECT entry, 6, resistance6, VerifiedBuild FROM creature_template WHERE resistance6 <> 0;
+
+INSERT INTO creature_template_spell SELECT entry, 0, spell1, VerifiedBuild FROM creature_template WHERE spell1 <> 0;
+INSERT INTO creature_template_spell SELECT entry, 1, spell2, VerifiedBuild FROM creature_template WHERE spell2 <> 0;
```

Comment:
```
ERROR 1054 (42S22) at line 281: Unknown column 'spell2' in 'field list'
```
Not sure what  travis-ci is doing here. Has `spell2` been removed already in master?

Reply:
maybe lack of ` on keys.

Reply:
Oh, I see what you mean now. The SQL file name is way old now, meaning travis-ci adds it to the DB before several newer update files being added to the test DB. causing the newer updates to fail. This is why it is bad to use a real date as name for the PR SQL files...

Please rename the SQL file to something like `2099_99_99_00_world.sql` or similar "way far into the future" numbers to avoid being overtaken by newer updates (and thereby making new updates (made after the PR started) fail in unpredictable ways.